pages/agent/v3/help/_bootstrap.md

```diff
@@ -65,13 +65,14 @@ See https://buildkite.com/docs/agent/v3/hooks for more details.
+<tr id="git-submodule-clone-config"><th><code>--git-submodule-clone-config value</code> <a class="Docs__attribute__link" href="#git-submodule-clone-config">#</a></th><td><p>Comma separated key=value git config pairs applied before git submodule clone commands. For example, `update --init`. If the config is needed to be applied to all git commands, supply it in a global git config file for the system that the agent runs in instead.<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_SUBMODULE_CLONE_CONFIG</code></p></td></tr>
 <tr id="git-mirrors-path"><th><code>--git-mirrors-path value</code> <a class="Docs__attribute__link" href="#git-mirrors-path">#</a></th><td><p>Path to where mirrors of git repositories are stored<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_MIRRORS_PATH</code></p></td></tr>
 <tr id="git-mirrors-lock-timeout"><th><code>--git-mirrors-lock-timeout value</code> <a class="Docs__attribute__link" href="#git-mirrors-lock-timeout">#</a></th><td><p>Seconds to lock a git mirror during clone, should exceed your longest checkout (default: 300)<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_MIRRORS_LOCK_TIMEOUT</code></p></td></tr>
 <tr id="git-mirrors-skip-update"><th><code>--git-mirrors-skip-update </code> <a class="Docs__attribute__link" href="#git-mirrors-skip-update">#</a></th><td><p>Skip updating the Git mirror<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_MIRRORS_SKIP_UPDATE</code></p></td></tr>
 <tr id="bin-path"><th><code>--bin-path value</code> <a class="Docs__attribute__link" href="#bin-path">#</a></th><td><p>Directory where the buildkite-agent binary lives<br /><strong>Environment variable</strong>: <code>$BUILDKITE_BIN_PATH</code></p></td></tr>
 <tr id="build-path"><th><code>--build-path value</code> <a class="Docs__attribute__link" href="#build-path">#</a></th><td><p>Directory where builds will be created<br /><strong>Environment variable</strong>: <code>$BUILDKITE_BUILD_PATH</code></p></td></tr>
 <tr id="hooks-path"><th><code>--hooks-path value</code> <a class="Docs__attribute__link" href="#hooks-path">#</a></th><td><p>Directory where the hook scripts are found<br /><strong>Environment variable</strong>: <code>$BUILDKITE_HOOKS_PATH</code></p></td></tr>
+<tr id="sockets-path"><th><code>--sockets-path value</code> <a class="Docs__attribute__link" href="#sockets-path">#</a></th><td><p>Directory where the agent places sockets (default: "/Users/ben/.buildkite-agent/sockets")<br /><strong>Environment variable</strong>: <code>$BUILDKITE_SOCKETS_PATH</code></p></td></tr>
```

Comment:
```suggestion
<tr id="sockets-path"><th><code>--sockets-path value</code> <a class="Docs__attribute__link" href="#sockets-path">#</a></th><td><p>Directory where the agent places sockets (default: "$HOME/.buildkite-agent/sockets")<br /><strong>Environment variable</strong>: <code>$BUILDKITE_SOCKETS_PATH</code></p></td></tr>
```

I think we should hard code the value to use `$HOME` in https://github.com/buildkite/agent/blob/ac3643bdd906ae422398094708cab09a0e324db2/clicommand/bootstrap.go#L276

Reply:
And whatever it is on windows...